examples/api/cpp/floating_point_arith.cpp

```diff
@@ -73,7 +73,7 @@ int main()
   // different rounding modes.
   Term rtp = solver.mkRoundingMode(ROUND_TOWARD_POSITIVE);
   Term rtn = solver.mkRoundingMode(ROUND_TOWARD_NEGATIVE);
-  Op op = solver.mkOp(FLOATINGPOINT_TO_SBV, 16);  // (_ fp.to_sbv 16)
```

Comment:
Does this work? I seem to remember there was an issue with overload resolution and the `std::vector<T>(size_t)` constructor...

Reply:
Yes, it works because the initializer list takes precedence over the vector constructor.